CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ proExecuteStep()
 
 if(${XP_STEP} STREQUAL "build")
 # move some needed cmake files into place
-configure_file(Findovsrpro.cmake ${STAGE_DIR}/share/cmake/Findovsrpro.cmake COPYONLY)
+#configure_file(Findovsrpro.cmake ${STAGE_DIR}/share/cmake/Findovsrpro.cmake COPYONLY)
```

Comment:
(question) I'm guessing that since you've commented this out and renamed `Findovsrpro.cmake` to `Findscript.cmake.in` that this line could just be removed?